src/styles.scss

```diff
@@ -32,7 +32,7 @@ app-root {
 .mat-stroked-button,
 .mat-raised-button {
   text-transform: uppercase;
-  border-radius: 4px;
+  border-radius: 4px !important;
```

Comment:
Please change selector to be more accurate to avoid using !important.

Reply:
That's exact the same selector as in Angular Material. In order to override buttons without `!important` we need to add selectors like `.mat-flat-button.mat-primary` `.mat-flat-button.mat-accent` and for every colour available. I would rather use `!important` than compound selectors.